CommonTools/RecoUtils/src/precompile.h

```diff
@@ -0,0 +1,13 @@
+// for integration test 
```

Comment:
is PAT

#include "DataFormats/MuonReco/interface/Muon.h"
+
+#include "DataFormats/JetReco/interface/BasicJetCollection.h" 
+#include "DataFormats/JetReco/interface/CaloJetCollection.h" 
+#include "DataFormats/JetReco/interface/JPTJetCollection.h"
+#include "DataFormats/JetReco/interface/GenJetCollection.h"
+#include "DataFormats/JetReco/interface/PFJetCollection.h"
+#include "DataFormats/JetReco/interface/GenericJetCollection.h"
+#include "DataFormats/JetReco/interface/JetTrackMatch.h"


-rw-r--r--. 1 innocent zh 138977873 Jan 30 17:37 ../include/slc6_amd64_gcc491/CommonTools/CandUtils/src/precompile.h.gch
-rw-r--r--. 1 innocent zh 174274591 Feb  3 13:08 ../include/slc6_amd64_gcc491/CommonTools/RecoUtils/src/precompile.h.gch
-